GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -266,20 +287,23 @@ private void ReloadCommitInfo()
 
             void UpdateCommitMessage()
             {
+                if (_revision == null)
+                {
+                    rtbxCommitMessage.SetXHTMLText(string.Empty);
+                    return;
+                }
+
                 var data = _commitDataManager.CreateFromRevision(_revision, _children);
 
-                if (_revision != null && (_revision.Body == null || (AppSettings.ShowGitNotes && !_revision.HasNotes)))
+                if (_revision.Body == null || (AppSettings.ShowGitNotes && !_revision.HasNotes))
                 {
                     _commitDataManager.UpdateBody(data, out _);
                     _revision.Body = data.Body;
                     _revision.HasNotes = true;
                 }
 
                 var commitMessage = _commitDataBodyRenderer.Render(data, showRevisionsAsLinks: CommandClickedEvent != null);
-
-                // workaround the problem that with some DPI RichTextBox size height
-                // passed to ContentResized event is ~1 line less than needed
-                rtbxCommitMessage.SetXHTMLText(commitMessage + Environment.NewLine);
+                rtbxCommitMessage.SetXHTMLText(commitMessage);
```

Comment:
Are you sure, this will not cause a regression?

Reply:
I added this in 3.0 for 150% scaling.
This shouldn't cause any regressions

Reply:
Do I understand correctly that this workaround is not needed anymore?

Reply:
I don't think so. I don't believe we should have it.

I added it for 150% scaling because the textbox control felt squashed at the bottom. At 100% and 200% scaling the textbox has the correct margin.
However having run it for the past 4-5 months, I don't think it was a good addition.